lib/src/components/NxProgressBar/__tests__/NxProgressBar.test.tsx

```diff
@@ -0,0 +1,121 @@
+        expect(getShallowComponent({ value: 0 }).find(progressSelector)).toHaveProp('max', 100);
+        expect(getShallowComponent({ value: 0, max: 50 }).find(progressSelector)).toHaveProp('max', 50);
+        expect(getShallowComponent({ value: 20, max: 40 }).find(progressSelector)).toHaveProp('value', 20);
+      }
+  );
+
+  it('displays the correct className and hides label when inlineCounter is set to true',
+      function() {
+        const component = getShallowComponent({ value: 100, label: 'hello' });
+        const inlineCounterComponent = getShallowComponent({ value: 100, inlineCounter: true, label: 'hello' });
+
+        const labelTextSelector = '.nx-progress-bar__label-text';
+
+        expect(component).not.toHaveClassName('nx-progress-bar--inline-counter');
+        expect(component.find(labelTextSelector)).toExist();
+        expect(component.find(labelTextSelector)).toHaveText('hello');
+
+        expect(inlineCounterComponent).toHaveClassName('nx-progress-bar--inline-counter');
+        expect(inlineCounterComponent.find(labelTextSelector)).not.toExist();
+      }
+  );
+
+  it('sets the correct counter percentage value',
+      function() {
+        const counterSelector = '.nx-counter';
+
+        expect(getShallowComponent({ value: 20, max: 40 }).find(counterSelector)).toHaveText('50%');
+        expect(getShallowComponent({ value: 0 }).find(counterSelector)).toHaveText('0%');
+        expect(getShallowComponent({ value: 50 }).find(counterSelector)).toHaveText('50%');
+        expect(getShallowComponent({ value: 100 }).find(counterSelector)).toHaveText('100%');
+      }
+  );
+
+  it('sets the correct success className when progress is 100%',
+      function() {
+        const component = getShallowComponent({ value: 0 });
+        const successComponent = getShallowComponent({ value: 100 });
+        expect(component).not.toHaveClassName('nx-progress-bar--success');
+        expect(successComponent).toHaveClassName('nx-progress-bar--success');
+      }
+  );
+
+  it('shows the correct label and icon when progress is 100%',
+      function() {
+        const component = getShallowComponent({ value: 100, label: 'normal label' });
+        const successLabelComponent = getShallowComponent({ value: 100, label: 'success label' });
+
+        const labelTextSelector = '.nx-progress-bar__label-text';
+
+        expect(component).toContainMatchingElement(NxFontAwesomeIcon);
+        expect(component.find(labelTextSelector)).toHaveText('normal label');
+
+        expect(successLabelComponent).toContainMatchingElement(NxFontAwesomeIcon);
+        expect(successLabelComponent.find(labelTextSelector)).toHaveText('success label');
+      }
+  );
+
+  it('sets the correct error className when hasError is true',
+      function() {
+        const component = getShallowComponent({ value: 100 });
+        const errorComponent = getShallowComponent({ value: 100, hasError: true });
```

Comment:
Why is `value` being set in this test?

Reply:
> Why is `value` being set in this test?

Ops, removed it as part of this commit: 40650085762ac5ec62e6e5927255bc82dc399347